tests/unit/javascript/conftest.py

```diff
@@ -134,7 +176,98 @@ def run(self, source):
         return self.webview.page().mainFrame().evaluateJavaScript(source)
 
 
+class JSWebEngineTester:
```

Comment:
I wonder, could some of the things here be in a common baseclass for `JSTester` and this, to reduce code duplication?